src/Api/Vault/Controllers/SecurityTaskController.cs

```diff
@@ -87,6 +105,37 @@ public async Task<ListResponseModel<SecurityTasksResponseModel>> BulkCreateTasks
         [FromBody] BulkCreateSecurityTasksRequestModel model)
     {
         var securityTasks = await _createManyTasksCommand.CreateAsync(orgId, model.Tasks);
+
+        var securityTaskCiphers = await _getSecurityTasksNotificationDetailsQuery.GetNotificationDetailsByManyIds(orgId, securityTasks);
+
+        // Get the number of tasks for each user
+        var userTaskCount = securityTaskCiphers.GroupBy(x => x.UserId).Select(x => new
+        UserSecurityTasksCount
+        {
+            UserId = x.Key,
+            Email = x.First().Email,
+            TaskCount = x.Count()
+        }).ToList();
+
+        var organization = await _organizationRepository.GetByIdAsync(orgId);
+        await _mailService.SendBulkSecurityTaskNotificationsAsync(organization.Name, userTaskCount);
+
+        foreach (var userSecurityTaskCipher in securityTaskCiphers)
+        {
+            // Get the associated security task for the cipher
+            var securityTask = securityTasks.First(x => x.CipherId == userSecurityTaskCipher.CipherId);
+            // Create a notification for the user with the associated task
+            var notification = new Notification
+            {
+                UserId = userSecurityTaskCipher.UserId,
+                OrganizationId = orgId,
+                Priority = Priority.Informational,
+                ClientType = ClientType.Browser,
+                TaskId = securityTask.Id
+            };
+            await _createNotificationCommand.CreateAsync(notification);
```

Comment:
🎨  We'll need to tweak this command to avoid "pushing" every created notification per user out to the clients. We could probably add an optional argument like `bool skipPush = false` to the `CreateAsync()` signature and have it skip the call to the `_pushNotificationService.PushNotificationAsync`.

💭 I think we could get away with only sending the push notification for the last notification created per user. Because, once they get the push notification, the client will be instructed to fetch all the tasks. _**But**_, there may be other drawbacks and maybe we do need to explicitly push a "bulk" notifications created payload.